BMM.Core/Implementations/ApiClients/HttpHeaderProviders.cs

```diff
@@ -57,5 +57,18 @@ public IList<IHeaderProvider> GetProviders()
                 };
             }
         }
+        
+        public class AndroidMediaRequests : IPredefinedHeaderSet
```

Comment:
We should explain in a comment somewhere why this is needed. (Can't do async in ExoPlayer when trying to play a track. Not a problem since the token should have been refreshed by that point anyways ....)